scripts/build-circt.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+set -eu
+
+GIT_COMMIT=093cdfe482530623fea01e1d3242af93e533ba54
+
+CIRCT_BUILD=${PWD}/build-circt
```

Comment:
I'm realizing there might already be quite a few places where we assume that file paths don't contain spaces. It might be better to just get this PR through first, then do a pure cleanup PR by testing everything in a folder with spaces, (if we care?)